client/client-multi/client-c/Modules/doxygen/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This cmake list is resposible for building doxygen documentation.
+
+include (FindDoxygen)
+
+if(DOXYGEN_FOUND)
+
+   add_custom_target(doxygen
```

Comment:
You should create a directory for documentation before calling doxygen as it can't create them without help.